### ISSUE TITLE
Unexpected NNCF logs

### ISSUE THREAD

**Issue by kshpv** [openvinotoolkit/nncf#1032]
What is the reason for logging the message - "Please, provide execution parameters for optimal model initialization"? When I run the code below I see this message. It could confuse the users.

```
nncf_config_dict = {
    "input_info": {
        "sample_size": [1, 3, *IMAGE_SIZE]
    },
    "log_dir": str(OUTPUT_DIR),
    "compression": {
        "algorithm": "quantization",
        "initializer": {
            "range": {
                "num_init_samples": 15000
            },
            "batchnorm_adaptation": {
                "num_bn_adaptation_samples": 4000
            }
        }
    }
}

nncf_config = NNCFConfig.from_dict(nncf_config_dict)
nncf_config = register_default_init_args(nncf_config, train_loader)
```
Output:
`INFO:nncf:Please, provide execution parameters for optimal model initialization`

**Comment by vshampor** [comment]
Fixed in #1418

**Comment by kshpv** [comment]
> Fixed in #1418

Thanks! It's worth the wait!
